### PR TITLE
601 display auto assign attribute

### PR DESCRIPTION
**Pull request by MattW216**
Hi, @kgf Is this what you were looking for? Or would you like it in a different location? Let me know.

### CODE REVIEW

File: config/database.example.yml [hunk 1]
```diff
@@ -3,21 +3,22 @@ development:
   encoding: unicode
   database: lims_development
   pool: 5
-  username:
-  password:
+  username: <%= ENV['matt'] %>
```
Comment (kgf) [inline config/database.example.yml]: I will not want any of the changes to this file...
And you may want to change your password!

**Comment by kgf** [comment]
Thanks for your help, comments made.

**Comment by MattW216** [comment]
I'm sure it's just me, but I've just looked through each of the commits above, and at the conversation on the Issue, but I don't see any comments...

**Comment by kgf** [comment]
Sorry, GitHub had it as a pending review, instead of a simple comment. You should be able to see it now.

**Comment by MattW216** [comment]
Should be all set. Sorry about that - misunderstood the set-up instructions.

**Comment by kgf** [comment]
Thanks for your help, @MattW216 
Fixes #601

**Pull request merged**